plans/backup.pp

```diff
@@ -0,0 +1,66 @@
+# @summary Backup the core user settings for puppet infrastructure
+#
+# This plan can backup data as outlined at insert doc 
+# 
+plan peadm::backup (
+  # Standard
+  Peadm::SingleTargetSpec           $primary_host,
+  Optional[Peadm::SingleTargetSpec] $replica_host            = undef,
+
+  # Large
+  Optional[TargetSpec]              $compiler_hosts          = undef,
+
+  # Extra Large
+  Optional[Peadm::SingleTargetSpec] $primary_postgresql_host = undef,
+  Optional[Peadm::SingleTargetSpec] $replica_postgresql_host = undef,
+
+  # Which data to backup
+  Boolean                            $backup_orchestrator    = true,
+  Boolean                            $backup_rbac            = true,
+  Boolean                            $backup_activity        = true,
+  Boolean                            $backup_ca_ssl          = true,
+  Boolean                            $backup_puppetdb        = false,
+  Boolean                            $backup_classification  = true,
+  String                             $backup_directory       = '/tmp'
```

Comment:
UX: it seems like we're using `backup_*` as a parameter prefix indicating a kind of thing to backup or not to backup. Therefore, we should not use `backup_*` as the prefix for this parameter name, which is something different. This parameter defines where to place the backup artifact.

How about `output_directory`?

```suggestion
  String                             $output_directory       = '/tmp',
```